membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -74,15 +74,24 @@ class AttributeController extends Controller with LoggingWithLogstashFields {
         case Some(identityId) =>
           val (fromWhere, attributes) = pickAttributes(identityId)
           attributes.map {
-            case Some(attrs @ Attributes(_, Some(tier), _, _, _, _, _)) =>
-              log.info(s"$identityId is a member - $endpointDescription - $attrs found via $fromWhere")
+            case Some(attrs @ Attributes(_, Some(tier), _, _, _, _, _, _)) =>
+              log.info(s"$identityId is a $tier member - $endpointDescription - $attrs found via $fromWhere")
               onSuccessMember(attrs).withHeaders(
                 "X-Gu-Membership-Tier" -> tier,
                 "X-Gu-Membership-Is-Paid-Tier" -> attrs.isPaidTier.toString
               )
+            case Some(attrs @ Attributes(_, _, _, _, _, _, _, Some(date))) =>
+              log.info(s"$identityId is a digital subscriber expiring $date - $endpointDescription - found via $fromWhere")
+              onSuccessSupporter(attrs)
```

Comment:
just put the logging in the pattern match, and don't repeat the onSuccessSupporter(attrs) bit